### DISCUSSION TITLE
Is it possible to define the config file location

### DISCUSSION THREAD

**Discussion by Oak-Trees** [RaspAP/raspap-docker#37] [Q&A] [answered]
Is it possible to run RaspAP in a Docker container and tell it to use the config file in the /etc/ folder or can only the Docker yaml (or what ever it's called) file be use to configure the Docker version of RaspAP?

**Comment by billz** [comment, marked as answer]
You can access files in the host system with [bind mounts and volumes](https://docs.docker.com/guides/walkthroughs/access-local-folder/). Although it might be easier to simply [allocate a terminal](https://docs.raspap.com/docker/#allocating-a-terminal) and modify the networking defaults this way.

**Reply by Oak-Trees** [reply]
Thank you so much. I'm still not sure about everything that you mean, but I'll give it both a try.
